connexion/security.py

```diff
@@ -403,6 +523,126 @@ async def wrapper(request, token, required_scopes):
 
         return wrapper
 
+
+class OIDCSecurityHandlerFactory(AbstractSecurityHandlerFactory):
+    """
+    SecurityHandlerFactory for the OIDC security scheme.
+    """
+
+    # TODO
+
+
+SECURITY_HANDLERS = {
```

Comment:
Just to confirm if I understand this. If I want to add a custom `SecurityHandlerFactory` for an http scheme with "digest" type, I just add `"digest": DigestSecurityHandlerFactory` as a top level key here, right?

Reply:
Yes, indeed. Not entirely convinced that this is the best approach, but seemed to be the best starting point.

Happy to hear if you have other ideas :)

Reply:
As long as we don't expect any conflicts, I think this is fine. I had a quick look, and as far as I can tell there are no conflicts within the standards officially supported by OpenAPI.